### ISSUE TITLE
Ensure WebView is properly disposed after use

### ISSUE THREAD

**Issue by mikehardy** [ankidroid/Anki-Android#5175]
It would be better if the WebViews were actually re-used, but in the meantime, they should at least be properly disposed. While triaging a Chrome crash the responding Chromium developer came to the quick-n-dirty (thus maybe not accurate) conclusion that we weren't disposing the WebViews specifically (which could be bad because the Java-level GC is unaware of the native allocations so may not feel it needs to dispose as quickly as it does), and also that even when we do dispose the WebView we may not be doing it correctly.

More info on this comment: https://bugs.chromium.org/p/chromium/issues/detail?id=916779#c11

**Comment by mikehardy** [comment]
I examined this and the results were somewhat interesting.

1. we are re-using the WebView for all sequential uses of card question or answer display within the same Activity (Previewer, or Reviewer for instance).
1. the content is obliterated each time by calling WebVew.loadDataWithBaseURL() inside fillFlashCard(), so to the content it appears as though the card was obliterated but it's within the same WebView
1. the WebView is correctly detached from it's ViewGroup and WebView.destroy() is called via the destroyWebView() method in AbstractFlashcardViewer, so I think we are cleaning up correctly

Basically, this was a surprise because of the first item, but nice to see with regard to WebView hygiene.

On the negative side this means that if we're flipping through enough large images, then allowing large images in may be the source of the WebView crash I'm hunting.
